src/main/steps/common/yourhearings/hearings/template.njk

```diff
@@ -1,5 +1,6 @@
 {% extends "common/template.njk" %}
 {% extends "common/breadcrumb/template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
```

Comment:
Why govukTable is imported?

Reply:
Yeah sorry.. may be by mistake I have added this... I will remove this.